invoicer_project/static/js/clients_list_request.js

```diff
@@ -0,0 +1,86 @@
+const host = "http://127.0.0.1:8000";
+
+async function getUserData() {
+    let jsonResponse, response
+    try {
+        const headers = new Headers();
+        headers.append('Authorization', `Bearer ${window.localStorage.getItem('accessToken')}`);
+        const result = await fetch(host + '/clients/list/', {
+            method: "GET",
+            headers: headers,
+        })
+        response = result.status;
+        jsonResponse = await result.json();
+    } catch (error) {
+        console.error('Going to obtain new access token!');
+    }
+    return { 'responseStatus': response, 'data': jsonResponse }
+} 
+
+function fillInitials(userData) {
+     const userFirstName = userData["first_name"];
+     const userLastName = userData["last_name"];
+     document.getElementById("user_name").textContent = userFirstName + " " + userLastName;
+}
+
+function createClientListContent(data) {
+for(let i = 0; i < data.length; i++) {
+    document.getElementById("other_elements").insertAdjacentHTML('afterbegin', `<div class="row client_list_item align-items-center justify-content-between">
+                <div class="col-xxl-1 col-xl-1 col-1 list_item_user_icon_initials">
```

Comment:
We can't put this code in html because we don't know in advance how many customers there will be.
We receive the number of clients in the response from the server and based on this we create the appropriate number of these elements.